### PR TITLE
Worked on README

### PR DESCRIPTION
**Pull request by dmurtari**
Reverted back to the previous commit, so the only changes should be the changes that I made to the README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,18 +1,100 @@
...
+requires that your credentials from the cloud server that you are accessing be
+available. 
+
+For example, using the Nimbus cloud on [FutureGrid]:
+
+    ssh <username>@hotel.futuregrid.org
+    cd .nimbus
+    cat querytokens.sh
+
+Run the two export commands to the local machine.
```
Comment (tvoran) [inline README.md]: Should be "on the local machine".

File: README.md [hunk 2]
```diff
@@ -1,18 +1,100 @@
...
+Run the two export commands to the local machine. 
+
+Automaton should now be configured and ready to run.
+
+
+Running Automaton
+-----------------
+
+Deploying the Automaton software, executing the benchmarks, and gathering and 
+generating graphs is accomplished through the use of flags that can be seen be
```
Comment (tvoran) [inline README.md]: Should be "can be seen by".

File: README.md [hunk 3]
```diff
@@ -1,18 +1,100 @@
...
     python ./automaton.py -h
 
+If a help message is shown, then the required libraries have been installed 
+successfully. 
+
+Configuration & Environmental Variables
+---------------------------------------
+
+Automaton has a global configuration files that needs to be preconfigured before
+benchmarks can be run -- etc/global.conf. Because automaton uses ssh to connect
```
Comment (tvoran) [inline README.md]: Should probably capitalize Automaton here.

File: README.md [hunk 4]
```diff
@@ -1,18 +1,100 @@
...
+to servers, you must provide the path to your public and private keys. Assuming 
+you already have generated a key pair (if not, see Github's [guide to generating
+ssh keys]), the two lines that need to be edited are:
+
+    key_path = path/to/public/key
+    ssh_priv_key = path/to/private/key
+
+In addition to configuring etc/global.conf with your ssh keys, automaton also
```
Comment (tvoran) [inline README.md]: Capitalize Automaton.

File: README.md [hunk 5]
```diff
@@ -1,18 +1,100 @@
...
+
+Executing the benchmarks may take some time to complete, but once they are finished
+running the logs can be gathered with:
+
+    python automaton.py -o
+
+And finally the graphs generated with 
+
+    python automaton.py -p
+
```
Comment (tvoran) [inline README.md:97]: Might as well add mention of terminating the clusters here too, since it starts with launching the clusters.

**Comment by tvoran** [comment]
The README looks pretty good, see the minor comments I made. You should also remove etc/clouds.conf, etc/global.conf, and self.db from the pull request.

Also, can you squash your commit down to one commit that briefly describes what you did with the README? (By brief I mean a short sentence.) That'll make things cleaner.

**Comment by dmurtari** [comment]
Sounds good, I'll take care of that and squash the commit down to one.

**Comment by tvoran** [comment]
Looks good, thanks.

**Pull request merged**